### PR TITLE
fix: delete env from settings

### PR DESCRIPTION
**Pull request by devthejo**
fix https://github.com/bobheadxi/deployments/issues/118

### CODE REVIEW

**Comment by ai** [comment]
@bobheadxi need this PR a lot since the current deployment become slower and slower after every deploy

**Comment by devthejo** [comment]
just tested here: https://github.com/SocialGouv/deployments/actions/runs/3954292536 and it works nicely

File: README.md [hunk 1]
```diff
@@ -252,6 +252,8 @@ jobs:
 
 This is the same as `deactivate-env`, except deletes the environment entirely. See [`step: deactivate-env`](#step-deactivate-env) for more details.
 
+GITHUB_TOKEN doesn't not allow to delete environment, you have to set a personal access token and provide it as `token` option.
```
Comment (bobheadxi) [inline README.md:255]:
```suggestion
Note that the default `GITHUB_TOKEN` does not allow environment deletion - you have to set a personal access token and provide it in the `token` input.
```

**Review by bobheadxi — COMMENTED** [review]
Thank you for the fix! The change LGTM, other than a minor doc update - I can't commit to this branch, so please apply the suggested change and we can get this merged :)

**Comment by bobheadxi** [comment]
There is also an issue with the build: https://github.com/bobheadxi/deployments/actions/runs/3954512017/jobs/6771995515

**Comment by devthejo** [comment]
> There is also an issue with the build: https://github.com/bobheadxi/deployments/actions/runs/3954512017/jobs/6771995515

First I was making build with node v18 (default on my machine), but the updated workflow on my fork wasn't OK, so I've tested after nvm use 16 then it was passing, I've just tested with node 12, but no changes in diff, so I don't know what's the difference

**Comment by bobheadxi** [comment]
I upgraded deps/pipelines/etc and ran the rebuild, and cut a new release with this fix: https://github.com/bobheadxi/deployments/releases/tag/v1.4.0 , thanks!

**Pull request merged**